### DISCUSSION TITLE
live code editor like sandpack

### DISCUSSION THREAD

**Discussion by rahathosen** [code-hike/codehike#280] [Q&A] [answered]
code-hike work excellent for me and thanks to Rodrigo, now I want to live edit the code in code-hike like sandpack, is this possible?

**Comment by DanielFGray** [comment, marked as answer]
You can use sandpack and code-hike components in mdx

**Reply by rahathosen** [reply]
This approach i know but which i want to know that is it possible to edit live code on code-hike ?

**Reply by DanielFGray** [reply]
code-hike has no editable component, i raised this issue previously
https://github.com/code-hike/codehike/issues/197
